tests/test_claim_money_authority.py

```
@@ -0,0 +1,119 @@
+
+    assert result.price == 8_500_000
+    assert draft.price_of_claim == "8 500 000 тенге"
+    assert decision.mode == "property"
+    assert decision.amount == 255_000
+
+
+def test_production_regression_known_price_known_too_no_longer_becomes_unclassified() -> None:
+    install_claim_money_authority()
+    draft = _draft(
+        price="8 400 000 тенге",
+        requests=["Взыскать с Ответчика задолженность."],
```

Ruff (0.16.2)

[warning] String contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF001)